README.md

```diff
@@ -223,4 +340,43 @@ Response
   "deviceModel": "Samsung", // nullable
   "deviceType": "mobile" // mobile | desktop | web
 }
-```
\ No newline at end of file
+```
+
+## 7. Enums
+
+In this project, enums are used to define the possible values of some fields.
+
+### 7.1 Device Type
+
+Device types are only used for UI purposes like showing an icon.
+
+There is no difference in the protocol between the different device types.
+
+| Value    | Description                               |
+|----------|-------------------------------------------|
+| mobile   | mobile device (Android, iOS, FireOS)      |
+| desktop  | desktop (Windows, MacOS, Linux)           |
```

Comment:
typo error, `MacOS` trademark is `macOS`

![CleanShot 2023-04-06 at 10 18 29@2x](https://user-images.githubusercontent.com/3842474/230255845-dead075b-1cb7-4d0c-9cea-a994c3d6a087.png)

see https://www.apple.com/legal/intellectual-property/trademark/appletmlist.html

Reply:
Fixed